hasjob/views/helper.py

```diff
@@ -240,7 +240,7 @@ def record_views_and_events(response):
                     db.session.commit()
                 except IntegrityError:  # Race condition from parallel requests
                     db.session.rollback()
-            campaign_view_count_update.delay(campaign_id=campaign.id, user_id=g.user.id)
+            campaign_view_count_update.delay(datetime.utcnow(),campaign_id=campaign.id, user_id=g.user.id)
```

Comment:
Also, all parameters must be named parameters as you're calling a background job which runs in a different process, and there is the possibility that it will be running a different version of the code if this call happens during an upgrade process. Using named parameters will ensure the function fails if the receiving side doesn't yet accept this parameter.